card/mod/pointer/set/abstract/01_paging.rb

```diff
@@ -45,14 +45,14 @@ def paging_links
 
   # First page is 0 (not 1)
   def page_link_li text, page, status, options={}
-    wrap_with :li, class: status do
+    wrap_with :li, class: "page-item #{status}" do
       page_link text, page, options
     end
   end
 
   def page_link text, page, options
-    return text unless page
-    options.merge! class: "card-paging-link slotter",
+    return content_tag(:div, text.html_safe, class: "page-link") unless page
+    options.merge! class: "card-paging-link slotter page-link",
```

Comment:
it might be nice to change "page" to "paging", because that's less ambiguous (we use "page" to mean other things)